examples_gbs/run_tutorial_points.py

```diff
@@ -0,0 +1,147 @@
+# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports,invalid-name
+r"""
+Point Process Tutorial
+======================
+
+This tutorial shows how to generate GBS point process samples and use them to detect outlier
+points in a data set. Point processes are models for generating random point patterns and GBS
+devices can be programmed to operate as special types of point processes that generate clustered
+random point patterns :cite:`jahangiri2019point`.
+
+The probability of generating a specific pattern of points in GBS point processes depends on
+matrix functions of a kernel matrix that describes the similarity between the points. Matrix
+functions that appear in GBS point processes are typically
+`permanents <https://en.wikipedia.org/wiki/Permanent_(mathematics)>`__ and
+`hafnians <https://hafnian.readthedocs.io/en/latest/hafnian.html>`__. In this tutorial, we use the permanental point
```

Comment:
@josh146 
Should we be linking here or to `thewalrus`?
https://the-walrus.readthedocs.io/en/latest/hafnian.html
Also to the `latest` or `stable` versions?